cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -68,7 +68,8 @@ var (
 	enableLeaderElection = flag.Bool("leader-election", false, "Enables leader election. If leader election is enabled, additional RBAC rules are required. Please refer to the Kubernetes CSI documentation for instructions on setting up these RBAC rules.")
 
 	leaderElectionNamespace = flag.String("leader-election-namespace", "", "Namespace where the leader election resource lives. Defaults to the pod namespace if not set.")
-	strictTopology          = flag.Bool("strict-topology", false, "Passes only selected node topology to CreateVolume Request, unlike default behavior of passing aggregated cluster topologies that match with topology keys of the selected node.")
+	strictTopology          = flag.Bool("strict-topology", false, "Late binding: pass only selected node topology to CreateVolume Request, unlike default behavior of passing aggregated cluster topologies that match with topology keys of the selected node.")
+	immediateTopology       = flag.Bool("no-immediate-topology", true, "Immediate binding: pass aggregated cluster topologies for all nodes where the CSI driver is available (enabled, the default) or no topology requirements (if disabled).")
```

Comment:
Can we name the variable noImmediateTopology to match?

Reply:
I got stuck halfway here removing a double negation: `--no-immediate-topology=false` is what I wanted to avoid. I just missed updating the flag name, variable name and default are correct already. Will fix.

Reply:
Fixed. Only the flag name and documentation changed, the implementation is the same as before.